### PR TITLE
Added Filter to hide owner commands from all users

### PR DESCRIPTION
**Pull request by Dhruvin-Purohit**

### CODE REVIEW

**Review by XynoxTheDev — CHANGES_REQUESTED** [review]
@Dhruvin-Purohit, you can add one more filter ie. if the command executor is the owner then owner-only commands are visible to them.

File: src/commands/Utilities/help.js [hunk 1]
```diff
@@ -80,7 +80,7 @@ class HelpCommand extends Command {
                     this.client.user.displayAvatarURL()
                 );
 
-            for (const category of this.handler.categories.values()) {
+            for (const category of this.handler.categories.filter((categ) => categ.id != 'Bot Owner Only').values()) {//filter will hide owner commands from all users
```
Comment (Anish-Shobith) [inline src/commands/Utilities/help.js:83]: We don't wanna hide commands of a developer if the commands are ran by a developer right ?

Comment (Dhruvin-Purohit) [reply src/commands/Utilities/help.js:83]: so an if statement maybe? like a function for filter command?

**Review by XynoxTheDev — CHANGES_REQUESTED** [review]

**Comment by Anish-Shobith** [comment]
Closing because no response from the author! Will make our pr for it, thank you for the input!

**Pull request closed without merging**